docs/source/modin/supported/index_supported.rst

```diff
@@ -5,6 +5,8 @@ The following table is structured as follows: The first column contains the meth
 The second column is a flag for whether or not there is an implementation in Snowpark for
 the method in the left column.
 
+Currently, there is no MultiIndex support.
```

Comment:
Does this mean we don't support any operations on MultiIndex objects? We definitely do have APIs that return them, at least.

Reply:
Nope, we don't support any operations on MultiIndex since we don't have an object representing them. The lazy Index object we currently have is only supposed to be a single Index.

Reply:
I think this is still worth clarifying, since it might imply to users that DF/Series operations cannot return MultiIndex objects, even though they're just returning eager pandas.MultiIndex.

Reply:
I suppose I'll say that there is no lazy MultiIndex support. This Index object is supposed to only be a single Index but existing DataFrame and Series APIs may support native pandas MultiIndex operations.

Reply:
should be in now if you want to look at the text